docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,98 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
-
-## Example external application
-
-We'll be using [samltest.id](https://samltest.id/) to create a test consumer of
-Teleport's SAML identity provider.
-
-## Step 1/4. Configure a Teleport role with access to SAML service provider objects
-
-First, ensure you are logged into Teleport as a user that has permissions
-to read and modify `saml_idp_service_provider` objects. The default `editor` role
-has access to this already, but in case you are using a more customized configuration,
-create a role called `sp-manager.yaml` with the following contents:
-
+  Reference](./saml-reference.mdx) before proceeding.
+- User with preset `editor` role. We recommend creating a dedicated role to manage
+Service Provider.
+
+  <Admonition type="tip" title="Recommended">
+  <Details title="Create SAML IdP Service Provider role" closed>
```

Comment:
Not sure what others thought are, but I've always preferred information-dense text as opposed to fancy UI controls or other "noise" on the page. Putting a collapsible box inside an admonition feels like a lot of cognitive load. Can we simplify?